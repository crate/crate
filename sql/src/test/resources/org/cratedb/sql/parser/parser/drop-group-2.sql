DROP GROUP IF EXISTS t2