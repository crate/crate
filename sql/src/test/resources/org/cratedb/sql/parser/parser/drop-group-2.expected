org.cratedb.sql.parser.parser.DropGroupNode@47d62270
name: t2
statementType: DROP GROUP
existenceCheck: IF_EXISTS